modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -927,6 +933,42 @@ public void execute(Project project) {
 
 			});
 
+		String bundleChecksumMD5 = _getBundleChecksumMD5(project);
+
+		download.doLast(
```

Comment:
doLast happens during task execution.  We need this to happen during configuration.  the `verifyBundleTask.src` property can be set with a `new Callable<File>{}` so that the actual file will be called later.